docs/troubleshooting/clusterResourcePlacementScheduled.md

```diff
@@ -0,0 +1,199 @@
+# How can I debug when my CRP status is ClusterResourcePlacementScheduled condition status is set to false?
+The `ClusterResourcePlacementScheduled` condition is set to `false` when the scheduler cannot find all the clusters needed as specified by the scheduling policy.
+> Note: In addition, it may be helpful to look into the logs for the scheduler controller to get more information on why the scheduling failed.

```

Comment:
nit: scheduler is not a controller is a separate component maybe rename to scheduler framework